__tests__/components/Header.test.js

```diff
@@ -25,8 +25,11 @@ describe('<Header />', () => {
 
   describe('nav tabs', () => {
     describe('when no resource in state', () => {
-      it('displays 3 header tabs', () => {
-        expect(wrapper.find('ul.editor-navtabs NavLink').length).toBe(3)
+      it('displays 4 header tabs', () => {
```

Comment:
I thought the Browse tab is the same as the search tab, not a new tab.  Currently the browse tab indicates:
> This is a temporary facility until we have search implemented.

Reply:
I can remove browse. The mock up had "Search", so I didn't think we'd want to replace it yet, but I'm fine with that.